.buildkite/docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM rust:1.54
+
+# INSTALL NODE ============================================================================
+# Unfortunately can't use the friendly lts name here easily, because then it gets hard to add to the
+# path. We'll just have to update this as needed.
+ENV NODE_VERSION="v14.17.5"
```

Comment:
Could have used lts and changed the code below to:

`ENV PATH="$(dirname $(. $NVM_DIR/nvm.sh && nvm use 16 &> /dev/null && which node)):$PATH"`